File: springboot/open-api/src/main/java/com/openapi/domain/dto/ws/response/InstructionTtsItem.java
package com.openapi.domain.dto.ws.response;

import com.openapi.domain.constant.ws.Instruction;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * {@link InstructionListResponse#getInstructions()} 中 {@link Instruction#TTS} 列表项，
 * 在 {@link TtsDataResponse} 基础上增加编排序号。
 */
@EqualsAndHashCode(callSuper = true)
@Data
public class InstructionTtsItem extends TtsDataResponse {

    private Instruction type = Instruction.TTS;

    private Integer index;
}

File: app/android/app/src/main/java/com/magicvector/domain/dto/ws/response/InstructionTtsItem.kt
package com.magicvector.domain.dto.ws.response

import com.magicvector.domain.constant.ws.Instruction

/**
 * [InstructionListResponse.instructions] 中 [Instruction.TTS] 列表项，字段与 [TtsDataResponse] 一致并增加编排序号。
 */
data class InstructionTtsItem(
    val index: Int,
    val type: Instruction = Instruction.TTS,
    val base64AudioStream: String,
    val text: String = "",
    val agentId: String = "",
    val seq: String = "",
    val isLast: Boolean = false,
    val timestamp: Long = 0L,
)
